MoreLinq.Test/RepeatTest.cs

```diff
@@ -71,14 +71,31 @@ public void TestRepeatForeverBehavior()
         }
 
         /// <summary>
+        /// Verify applying Repeat without passing count produces a circular sequence
+        /// </summary>
+        [Test]
+        public void TestRepeatForeverBehaviorManyElementsList()
+        {
+            const int repeatCount = 30;
```

Comment:
This `const` serves no purpose. It's only used in one place so either inline it or use it to dynamically calculate the 300 sent to `Take` later.